src/runtime/omp/omp_hardware_manager.cpp

```diff
@@ -142,7 +142,11 @@ omp_hardware_context::get_property(device_uint_property prop) const {
     return std::numeric_limits<std::size_t>::max();
     break;
   case device_uint_property::max_group_size:
+#ifndef HIPSYCL_USE_ACCELERATED_CPU
```

Comment:
I think this is confusing. There are two different ways in which this macro is used: At compile time of hipSYCL, affecting the hipSYCL runtime and clang plugin, and secondly, set by syclcc if accelerated cpu is enabled. What happens if `syclcc` compiles without this macro because the user has disabled accelerated cpu, but the runtime and clang plugin have it baked in? Would it make sense to distinguish one macro the runtime and clang plugin whether there is *support*, and a second one to be used by `syclcc` whether it is *enabled*?